rest/route/reliability_test.go

```diff
@@ -402,3 +448,165 @@ func TestRunTestHandler(t *testing.T) {
 	lastDoc := sc.MockTaskReliabilityConnector.CachedTaskReliability[100]
 	assert.Equal(lastDoc.StartAtKey(), resp.Pages().Next.Key)
 }
+
+func setupEnv(ctx context.Context) (*mock.Environment, error) {
+	env := &mock.Environment{}
+
+	if err := env.Configure(ctx, "", nil); err != nil {
+		return nil, errors.WithStack(err)
+	}
+	return env, nil
+}
+
+func withSetupAndTeardown(t *testing.T, env evergreen.Environment, fn func()) {
+	require.NoError(t, db.ClearCollections(stats.DailyTaskStatsCollection))
+	defer func() {
+		assert.NoError(t, db.ClearCollections(stats.DailyTaskStatsCollection))
+	}()
+
+	fn()
+}
+
+func TestReliability(t *testing.T) {
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	for opName, opTests := range map[string]func(ctx context.Context, t *testing.T, env evergreen.Environment){
```

Comment:
you don't need to name variables in function type declarations.

Reply:
:+1: